### PR TITLE
Lab 27

### PR DESCRIPTION
**Pull request by talwaserman**
lab 27

### CODE REVIEW

File: lab27/src/StarWarsActor.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        onChange={e => setActorId(e.target.value)}
+      />
+      {data && (
+        <section className="star-wars">
+          <p> Name: {data.name}</p>
+          <p> height: {data.height}</p>
+          <p>
+            <u>Films:</u>
+          </p>
+          {data.films.map(film => (
```
Comment (ynonp) [inline lab27/src/StarWarsActor.js:22]: So we talked about this today - it's always a good idea to limit the number of results you show on screen (regardless of what the server returned)

File: lab27/src/StarWarsActor.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React, { Fragment, useState } from 'react';
+import useSWR from 'swr';
```
Comment (ynonp) [inline lab27/src/StarWarsActor.js:2]: Can you write your own?

File: lab27/src/Vimeo.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React, { useEffect } from 'react';
+import Player from '@vimeo/player';
+
+const Vimeo = ({ id = 19231868, width = 640 }) => {
+  let player = null;
+  useEffect(() => {
```
Comment (ynonp) [inline lab27/src/Vimeo.js:6]: What happens when a new id is passed as prop?

File: lab27/src/StarWarsActor.js (hunk 2 above)
Comment (talwaserman) [reply lab27/src/StarWarsActor.js:2]: yes but this was not the task here.

File: lab27/src/Vimeo.js (hunk 3 above)
Comment (talwaserman) [reply lab27/src/Vimeo.js:6]: i didn't make the video change, i know... wanted to exercise other things. once i have this component
using the props and / or useEffect to change the video is not difficult

File: lab27/src/StarWarsActor.js (hunk 2 above)
Comment (ynonp) [reply lab27/src/StarWarsActor.js:2]: I think it will make a cool bonus task

**Pull request merged**